.github/workflows/pr-merged.yml

```diff
@@ -0,0 +1,27 @@
+name: Telegram P/R Merged notification
+
+on:
+  pull_request:
+    types:
+      - closed
+            
+jobs:
+  if_merged: 
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
```

Comment:
do you need to checkout the code to send the telegram message?

Reply:
I dont think so. I just used an example from [this article](https://medium.com/@derekmcauley7/using-github-actions-to-run-a-telegram-bot-c91072a42a38)

[github example](
https://github.com/derekmcauley7/github-actions-pragmatic-programmer-bot)